packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -144,6 +146,50 @@ def test_if_we_have_capabilities(cluster):
     assert {'name': 'PACKAGE_MANAGEMENT'} in r.json()['capabilities']
 
 
+def test_cosmos_package_add(cluster):
+    def package_add_configured():
+        user_config = load_yaml("/opt/mesosphere/etc/user.config.yaml")
```

Comment:
I fixed it to load the JSON you enumerated above. I greped the code and didn't find an util function. @mellenburg Let know if I missed it.